### ISSUE TITLE
Outperform one-shot baseline with RL

### ISSUE THREAD

**Issue by mnoukhov** [mnoukhov/emergent-compete#4]
now that #3 is solved for a deterministic policy learned with backprop, try to do the same thing learning with RL

current A2C method only matches one-shot baseline 
![a2c-reward](https://user-images.githubusercontent.com/3391297/57803799-3cdc2100-7727-11e9-9fbe-a6a00602fc93.png)
![a2c-round](https://user-images.githubusercontent.com/3391297/57803806-406fa800-7727-11e9-9552-ad325bdab186.png)
but results across rounds look promising, test some hyperparameter tuning and see if we can't improve performance, otherwise maybe switch to DDPG?
